dataactvalidator/filestreaming/csvAbstractReader.py

```diff
@@ -207,7 +207,6 @@ def _getLine(self):
             self.packetCounter +=1
 
             #Get the current lines
-            packet = packet.decode("utf-8") if not self.isLocal else packet
             currentBytes = self.unprocessed + packet
```

Comment:
Yes that variable is rather poorly named, it is intended to hold a string, unprocessed is the part of the last packet after the final line break, which is also a string.